server/server/models/category.py

```diff
@@ -9,7 +9,7 @@ class Category(Base):
     """SQLAlchemy model for table categories"""
 
     __tablename__ = 'categories'
-    
+
```

Comment:
It's preferable to exclude changes from PR that are not to related to feature you implement. Different style fixing and code cleaning can be posted as separate PR.